docs/index.rst

```diff
@@ -10,8 +10,9 @@ Information about the GMT data server and mirrors can be found on the
 `GMT organization website <https://www.generic-mapping-tools.org/mirrors/>`_.
 
 **Note**: New datasets for vertical deflections, FAA errors, sea level, dynamic
-topography, distance to GSHHG coastline and the planetary data will be available
-when GMT 6.5 is released.
+topography, distance to GSHHG coastline and the planetary data are available
+now that GMT 6.5 is released. Also note that the highest resolution data for
+Mars, Mercury, Moon and Pluto requires GMT 6.5 or later.
```

Comment:
The note gives me the impression that these new datasets require GMT 6.5.0. Maybe we should just keep the last sentence?

Reply:
What I try t say is that all the resolutions that are in integer minute or degrees are available but things that  cannot be written that way (e.g., top resolution of Mars MOLA is 12.1468873601s cannot be read by 6.4 since its internal variable is only 8 characters, so that metadata record is skipped at op Mars resolution for such folks will be 15s.  So most of the data is available on 6.4 except 4, the top of Mars, Moon, Mercery, and Pluto).  How do you want to say this?

Reply:
I meant to change the note to:

```
.. notes::

   The highest resolution data for Mars, Mercury, Moon and Pluto requires GMT 6.5 or later.   
```

Reply:
OK, see if I got it write with adding the .. notes ..:: tag.